formats/shared/default_properties/notification.jsonnet

```diff
@@ -14,4 +14,40 @@
   payload_version: {
     "$ref": "#/definitions/payload_version",
   },
+  email_document_supertype: {
```

Comment:
Worth noting that email-alert-api also currently uses this value and `government_document_supertype` to match a piece of content with its subscriber list: https://github.com/alphagov/email-alert-api/blob/master/app/queries/subscriber_list_query.rb#L2

Reply:
👍 cool thanks, I suspected as much so we'll keep the supertypes in the notifications until that's fixed.

Reply:
As far as I know it's taking it directly from the notification, https://github.com/alphagov/email-alert-service/blob/394b64617689ecdd3b3ee285021ef284e41fc7e0/email_alert_service/models/major_change_message_processor.rb#L41 comes from RabbitMQ, and then it's used here https://github.com/alphagov/email-alert-service/blob/394b64617689ecdd3b3ee285021ef284e41fc7e0/email_alert_service/models/email_alert.rb#L34-L35 .

Reply:
Ah good, was confused for a minute 👍